plugins/woocommerce-blocks/assets/js/blocks/product-filters/utils.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { BlockInstance, getBlockTypes } from '@wordpress/blocks';
+
+/**
+ * Returns an array of allowed block names excluding the disallowedBlocks array.
+ *
+ * @param disallowedBlocks Array of block names to disallow.
+ * @return Array of allowed block names.
+ */
+export const getAllowedBlocks = ( disallowedBlocks: string[] ) => {
```

Comment:
I mentioned this above but maybe instead of `disallowedBlocks` we change it to `excludedBlocks`? So it reads like "Get me all the allowed blocks, excluding these ones" instead of "Get me all the allowed blocks disallowing these blocks".

Reply:
Make sense! I renamed the constant followed your suggestion. Thanks!

Reply:
I think we should change the argument here as well to be `excludedBlocks`. Also I think this utility function already exists here: https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/product-filter/utils.ts

So we should consolidate it.

Reply:
@dinhtungdu I am not sure you saw this comment?

Reply:
@roykho Sorry I missed this. We will eventually remove the `product-filter` block, so I didn't pay too much attention to it. I kept the `product-filter` block for the migration purpose, but it has done its job now, let me remove it to prevent confusion.